### ISSUE TITLE
For each package, on chocolatey.org/packages, show a list of dependents

### ISSUE THREAD

**Issue by darcyparker** [chocolatey/chocolatey#272]
The list of dependents would be generated based on query of all other packages to see if they have a dependency on the current package.

I suggested this in https://github.com/chrissie1/chocolatey-Explorer/issues/68 as well. But I think it would be useful on the main chocolatey.org/packages web site too.  (npmjs.org shows dependents and dependencies)

**Comment by alexchandel** [comment]
Homebrew is capable of showing both dependencies (`brew depends gcc`) and reverse dependencies (`brew uses gcc`) with ease.

**Comment by equinoxmatt** [comment]
Is this due at some point? Seems quite fundamental for a package manager.  npm is a good comparison, but a better one would be apt used on Ubuntu.  After all, chocolatey refers to apt on it's website.

You can find dependent packages on apt easily.

**Comment by ferventcoder** [comment]
IMHO managing dependencies is fundamental for a pkg mgr - getting info about what depends on a pkg is very handy, but I'd hardly argue fundamental. 

However I'd like to understand a use case that makes this fundamental.

**Comment by equinoxmatt** [comment]
Fundamental is probably a strong word.  However most package managers I have used have the ability to look at what package is depending on it.  

Use case is when you have a package installed and you are unsure if removing it would affect any other packages....

**Comment by ferventcoder** [comment]
Agreed it would be great to see who depends on a package, graphed locally and on a package source.

**Comment by alexchandel** [comment]
Homebrew also has two forms, `brew uses gcc`, which lists all dependent formulae, and `brew uses --installed gcc`, which only lists installed ones. Useful when you're fixing installed packages.
